mitodl/ocw-studio
# Pt/resource picker tabs

#### Pre-Flight checklist


#### What are the relevant tickets?
Closes https://github.com/mitodl/ocw-studio/issues/1419 (together with https://github.com/mitodl/ocw-hugo-projects/pull/211).

#### What's this PR do?
The tabs for the ResourcePicker dialog are now defined in the relevant OCW Studio config files, rather than hardcoded in the frontend.

#### How should this be manually tested?
**This needs to be tested together with https://github.com/mitodl/ocw-hugo-projects/pull/211.** Start OCW Studio locally, and then replace the Website Starters in Django admin for `ocw-www` and `ocw-course` with the updated ones from https://github.com/mitodl/ocw-hugo-projects/pull/211. Try renaming an existing tab and adding a new tab type; check that this works properly within the ResourcePicker menu.
